drivers/Watchdog.h

```diff
@@ -30,14 +30,23 @@
 namespace mbed {
 
 /** \addtogroup drivers */
-/** Hardware system timer that will reset the system in the case of system failures or
- *  malfunctions. There is only one instance in the system.
+/** A hardware watchdog timer that will reset the system in the case of system
+ * failures or malfunctions. If you fail to refresh the Watchdog periodically,
+ * it will reset the system after a set period of time.
+ *
+ * There is only one instance in the system. Use Watchdog::get_instance to
```

Comment:
Maybe something like `There is only one instance of the Watchdog class in the system, which directly maps to the hardware`